modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -107,111 +107,434 @@ Status description:: error: data exception - unsupported value. The provided val
+
+=== 22N53
+
+Status description:: error: data exception - invalid use of PROFILE. Cannot 'PROFILE' query before results are materialized.
+
+=== 22N54
+
+Status description:: error: data exception - invalid map. Multiple conflicting entries specified for `{ $mapKey }`.
+
+// === 22N55
+
+// Status description:: error: data exception - required key missing from map. Map requires key `{ $mapKey }` but was missing from field `{ $field }`.
+
+=== 22N56
+
+Status description:: error: data exception - protocol message length limit overflow. Protocol message length limit exceeded (limit: `{ $boltMsgLenLimit }`).
+
+// === 22N57
+
+// Status description:: error: data exception - invalid protocol type. Protocol type is invalid. Invalid number of struct components (received `{ $count1 }` but expected `{ $count2 }`).
+
+// === 22N58
+
+// Status description:: error: data exception - invalid spatial component. Cannot read the specified `{ $component }` component from `{ $value }`.
+
+// === 22N59
+
+// Status description:: error: data exception - token does not exist. The `{ $tokenType }` token with id `{ $tokenId }` does not exist.
+
+// === 22N62
+
+// Status description:: error: data exception - relationship type does not exist. The relationship type `{ $relType }` does not exist.
+
+// === 22N63
+
+// Status description:: error: data exception - property key does not exist. The property key `{ $propKey }` does not exist.
+
+// === 22N64
+
+// Status description:: error: data exception - constraint does not exist. The constraint `{ $constrDescrOrName }` does not exist.
+
+// === 22N65
+
+// Status description:: error: data exception - equivalent constraint already exists. An equivalent constraint already exists: `{ $constrDescrOrName }`.
+
+=== 22N66
+
+Status description:: error: data exception - conflicting constraint already exists. A conflicting constraint already exists: `{ $constrDescrOrName }`.
+
+=== 22N67
+
+Status description:: error: data exception - duplicated constraint name. A constraint with the same name already exists: `{ $constr }`.
+
+// === 22N68
+
+// Status description:: error: data exception - dependent constraint managed individually. Dependent constraints cannot be managed individually and must be managed together with its graph type.
+
+=== 22N69
+
+Status description:: error: data exception - index does not exist. The index specified by `{ $idxDescrOrName }` does not exist.
+
+// === 22N70
+
+// Status description:: error: data exception - equivalent index already exists. An equivalent index already exists: `{ $idxDescrOrName }`.
+
+=== 22N71
+
+Status description:: error: data exception - index with the same name already exists. An index with the same name already exists: `{ $idx }`.
+
+// === 22N72
+
+// Status description:: error: data exception - index required by existing constraint. A requested operation can not be performed on the specified index because the index is part of a constraint.
```

Comment:
My best guess is it's in a PR not yet merged or it's usage has been removed since it's not in error codes sheet either. I would check with @brunobuss who added it